### ISSUE TITLE
The 'tags' Command Does Not List Tags Located in the Parent Transaction

### ISSUE THREAD

**Issue by OptikSage** [ledger/ledger#532]
When running the tags and/or tags --values commands I don't see any tags that are in the parent transaction, only from the postings.

With These Transactions:

```
2018/01/01 * Transaction1
    ; :TAG1:
    ; TAG2: Value
    Account1                              100
    Account2

2018/01/01 * Transaction2
    Account1                              100 ; :TAG3:
    ; TAG4: Value
    Account2
           
```
Execute:
`$ ledger -f .\FILE.DAT tags`

Result:
```
TAG3
TAG4
```

Expected:
```
TAG1
TAG2
TAG3
TAG4
```

Version: Ledger 3.1.1-20150915
OS: Windows 10

**Comment by tbm** [comment]
Just for reference, I'm pretty sure this was originally reported in Bugzilla as BZ#1043

**Comment by tbm** [comment]
This is a duplicate of issue #1043.

**Comment by tbm** [comment]
This is fixed in 960f4e809f983263a061b40dd3cbd9408c5f8e1f
